src/EventExport/FileWriter/TabularDataEventFormatter.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace CultuurNet\UDB3\EventExport\FileWriter;
+
+class TabularDataEventFormatter
+{
+    /**
+     * @var string[]
+     */
+    protected $includedProperties;
+
+    protected $columns;
```

Comment:
Or move the columns info to a separate class that both the formatter and file writer can make use of?